app/src/main/java/com/arygm/quickfix/ui/elements/QuickFixDetailsdescription.kt

```diff
@@ -0,0 +1,157 @@
+package com.arygm.quickfix.ui.elements
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Check
+import androidx.compose.material.icons.filled.Star
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.arygm.quickfix.model.quickfix.QuickFix
+import com.arygm.quickfix.ui.theme.poppinsTypography
+
+@Composable
+fun QuickFixDetailsScreen(
+    quickFix: QuickFix,
+    onShowMoreToggle: (Boolean) -> Unit,
+    isExpanded: Boolean
+) {
+  BoxWithConstraints(
+      modifier =
+          Modifier.fillMaxWidth()
+              .background(MaterialTheme.colorScheme.surface, RoundedCornerShape(16.dp))
+              .padding(16.dp)) {
```

Comment:
Small absolute value here, but if the composable well handeled when it is called then it's ok I think